docs/docs/dev/cli-reference/cli/node/delete/index.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: delete
+---
+
+# Command: `node approve`
+
+The `bacalhau node delete` command offers administrations the ability to remove a node from the cluster using its unique identifier.
+
+## Description:
+
+Using the `delete` sub-command, administrators can remove a node from the list of available compute nodes in the cluster. This feature is necessary for the management of the infrastructure.
+
+## Usage:
+
+```bash
+bacalhau node delete [id] [flags]
+```
+
+## Flags:
+
+- `[id]`:
+
+  - The unique identifier of the node you wish to describe.
+
+- `-h`, `--help`:
+
+  - Displays the help documentation for the `describe` command.
+
+- `-m message`:
+
+  - A message to be attached to the deletion action.
+
+## Global Flags:
+
+- `--api-host string`:
+
+  - Specifies the host for client-server communication through REST. This flag is overridden if the `BACALHAU_API_HOST` environment variable is set.
+  - Default: `"bootstrap.production.bacalhau.org"`
+
+- `--api-port int`:
+
+  - Designates the port for REST-based communication between client and server. This flag is overlooked if the `BACALHAU_API_PORT` environment variable is defined.
+  - Default: `1234`
+
+- `--log-mode logging-mode`:
+
+  - Determines the log format preference.
+  - Options: `'default','station','json','combined','event'`
+  - Default: `'default'`
+
+- `--repo string`:
+  - Points to the bacalhau repository's path.
+  - Default: `"`$HOME/.bacalhau"`
```

Comment:
I rather not have these repeated here and just document them one, optionally providing a link, otherwise it draw attention away from what this page is documenting imo.